go/vt/vttablet/tabletserver/query_engine_test.go

```diff
@@ -266,3 +266,15 @@ func newTestQueryEngine(queryPlanCacheSize int, idleTimeout time.Duration, stric
 	qe.InitDBConfig(dbcfgs)
 	return qe
 }
+
+func TestRedactSQLStatements(t *testing.T) {
```

Comment:
IMO this is not a particularly useful unit test that doesn't really show that this change is working end to end.

Better would be to piggy-back on the actual consolidation tests that you modified in #3675 so that after exercising the consolidation, you get the result of the /debug/consolidations UI to show that it did in fact redact the queries as expected.

As a template you could look at https://github.com/tinyspeck/vitess/blame/master/go/vt/vttablet/tabletserver/txlogz_test.go#L44-L51

Reply:
👍 Added unit tests that test the HTTP endpoint.